aimbrain/aimbrain.py

```diff
@@ -3,6 +3,7 @@
 
 Usage:
   aimbrain-cli auth (face|voice) <biometrics> --user-id=<uid> --api-key=<api_key> --secret=<secret> [--token=<token>] [--api-url=<api_url>] [--device=<device>] [--system=<system>]
+  aimbrain-cli behavioural-submit --user-id=<uid> --api-key=<api_key> --secret=<secret> --data=<data> [--api-url=<api_url>] [--device=<device>] [--system=<system>]
```

Comment:
Just had a thought, if we change this line to something like:

```
aimbrain-cli behavioural-submit <data> --user-id=<uid> --api-key=<api_key> --secret=<secret> [--api-url=<api_url>] [--device=<device>] [--system=<system>]
```

It will be slightly more in line with the other commands

Reply:
Updated in 58b9c07.